docs/resources/equinix_network_device.md

```diff
@@ -43,6 +43,7 @@ resource "equinix_network_device" "csr1000v-ha" {
   metro_code      = data.equinix_network_account.dc.metro_code
   type_code       = "CSR1000V"
   self_managed    = false
+  connectivity    = INTERNET-ACCESS
```

Comment:
The values for `connectivity` in examples should be wrapped in quotes, since the field is a string.
```suggestion
  connectivity    = "INTERNET-ACCESS"
```